docs/docs/other-css-frameworks.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Other CSS Frameworks
+---
+
+This guide introduces some other popular styling libraries that you might want to use in your project.
+
+## How these frameworks were chosen
+
+The following criteria were used to determine which frameworks to include in this guide:
+
+- It must work with React and Gatsby.
+- It must impact the styling of a Gatsby site.
+- It should be a commonly requested or searched in the Gatsby docs.
+- It must consider accessibility and not promote UI anti-patterns.
+
+## Bootstrap
+
+Bootstrap is a widely used CSS and JS framework that was created at Twitter in 2010. It gained popularity for its support of responsive and mobile-first design.
+
+Under the hood, Bootstrap uses [Sass](https://sass-lang.com/). Unlike other frameworks in this guide, Bootstrap does not provide React components directly. Instead, it provides a set of pre-built CSS class names like `"btn-primary"` or `"dropdown"` that you can use to style HTML elements.
+
+If you're using Bootstrap v4 or older, many of the JavaScript components will need you to also install jQuery to work correctly. As of [Bootstrap v5](https://blog.getbootstrap.com/2020/06/16/bootstrap-5-alpha/), the framework no longer depends on jQuery.
```

Comment:
The wording here feels a little awkward to me. Let me know what you think about this suggestion? I'd like to put more focus on what someone would need to _do_.

```suggestion
If you're using Bootstrap v4 or older, you'll also need to include jQuery to get many of the JavaScript components to work correctly. As of [Bootstrap v5](https://blog.getbootstrap.com/2020/06/16/bootstrap-5-alpha/), the framework no longer depends on jQuery.
```

Reply:
Lots of good background in this section! I didn't even realize they'd switched to Sass!